tests/views/test_api.py

```diff
@@ -34,18 +34,20 @@ class APIViewsTest(BaseTestCase):
     def setUp(self):
         self.app = self.create_app()
         self.client = self.app.test_client()
-        DB.create_all()
+        # only run the following few lines once
+        # DB.create_all()
+        # DB.session.add_all(self.starting_data)
+        # DB.session.commit()
 
     def tearDown(self):
-        DB.session.remove()
-        DB.drop_all()
+        # DB.session.close()
+        # DB.drop_all()
+        # DB.session.remove()
+        pass
 
     # **********  meta  *********
     def test_meta(self):
         """Is the meta object being generated?"""
-        DB.session.add_all(self.starting_data)
```

Comment:
Pretty sure the answer to that is yes, because it's added to the session and not the DB for each individual test. So they act independently. If you find that's not the case for your tests (that modify the db) ping me and I'll take a look if you don't feel like it